frontend/apps/picasso/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "node_modules/.bin/next dev",
-    "build": "node_modules/.bin/next build",
```

Comment:
Pref using  `node_modules/.bin/next` instead of global values which might cause issues with inconsistent build if the user has it installed globally.

Reply:
should i make it `npx next dev` and `npx next build` respectively